### PR TITLE
feat: add format tool

### LINKED ISSUE #884: Makefile: add a formatting tool
We may also need a unified formatting tool (just a suggestion).
Example script:
```bash
#!/bin/bash

set -e

go install github.com/incu6us/goimports-reviser/v2@latest

PROJECT_NAME=github.com/apache/apisix-ingress-controller
while IFS= read -r -d '' file; do
  goimports-reviser  -file-path "$file" -project-name $PROJECT_NAME
done <   <(find . -name '*.go' -not -path "./test/*" -print0)


PROJECT_NAME=github.com/apache/apisix-ingress-controller/test/e2e
while IFS= read -r -d '' file; do
  goimports-reviser  -file-path "$file" -project-name $PROJECT_NAME
done <   <(find . -name '*.go' -path "./test/*" -print0)
```

_Originally posted by @lingsamuel in https://github.com/apache/apisix-ingress-controller/issues/879#issuecomment-1047597239_

### PR DESCRIPTION
**Pull request by mangoGoForward**
Signed-off-by: mango <xu.weiKyrie@foxmail.com>

### Type of change:

- [ ] Bugfix
- [x] New feature provided
- [ ] Improve performance
- [ ] Backport patches

### What this PR does / why we need it:

Add a formatting tool and format project files. Resolved #884 

### Pre-submission checklist:

* [x] Did you explain what problem does this PR solve? Or what new features have been added?
* [ ] Have you added corresponding test cases?
* [ ] Have you modified the corresponding document?
* [ ] Is this PR backward compatible? **If it is not backward compatible, please discuss on the [mailing list](https://github.com/apache/apisix-ingress-controller#community) first**

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -61,7 +61,7 @@ lint:
 ### gofmt:                Format all go codes
 .PHONY: gofmt
```
Comment (lingsamuel) [inline Makefile:62]: Can we use a more consistent name, such as `update-gofmt`?

**Review by lingsamuel — APPROVED** [review]

File: Makefile (hunk 1 above)
Comment (mangoGoForward) [reply Makefile]: Thanks, Done in [aac3c65](https://github.com/apache/apisix-ingress-controller/pull/885/commits/aac3c6529b1164267f8f27f7cee8a357839b2496) commit.

**Comment by mangoGoForward** [comment]
Hi @lingsamuel ,`lint-checker-ci` failed, and some hyperlink got status `500`, maybe we should fix first.

**Comment by mangoGoForward** [comment]
> Hi @lingsamuel ,`lint-checker-ci` failed, and some hyperlink got status `500`, maybe we should fix first.

But I can link to those websites.

**Review by tao12345666333 — APPROVED** [review]
LGTM 
thanks

**Pull request merged**